.update-copyright.conf

```diff
@@ -0,0 +1,17 @@
+[project]
+name: lesson-template
```

Comment:
On Sun, Mar 15, 2015 at 07:10:22PM -0700, Greg Wilson wrote:
> > @@ -0,0 +1,17 @@
> > +[project]
> > +name: lesson-template
> 
> Does this need to be edited when it's copied into other lesson
> repositories?

Yes, if they also want to use update-copyright.py to maintain their
own author list.  Alternatively, they could delete
.update-copyright.conf and do their own thing.